File: java/Disk.java
class Disk{
	// atribut private
    private String type;
	private String capacity;
	private String price;

	// constructor
	public Disk(){}
	public Disk(String type, String capacity, String price){
		this.type = type;
		this.capacity = capacity;
		this.price = price;
	}

	// setter/getter
	public void settype(String type) {
		this.type = type;
	}
	public String gettype() {
		return type;
	}
	public void setcapacity(String capacity) {
		this.capacity = capacity;
	}
	public String getcapacity() {
		return capacity;
	}
	public void setprice(String price) {
		this.price = price;
	}
	public String getprice() {
		return price;
	}

	/*Method*/
	public void printDisk(){
		System.out.println("Disk type : " + this.type);
		System.out.println("Disk capacity : " + this.capacity);
		System.out.println("Disk price : " + this.price);
	}
}

File: python/Disk.py
class Disk:
	# atribut private
	__type = ""
	__capacity = ""
	__price = ""

	# constructor
	def __init__(self, type, capacity, price):
		self.__type = type
		self.__capacity = capacity
		self.__price = price

	# getter/setter
	def settype(self, type):
		self.__type = type

	def gettype(self):
		return self.__type

	def setcapacity(self, capacity):
		self.__capacity = capacity

	def getcapacity(self):
		return self.__capacity
	
	def setprice(self, price):
		self.__price = price

	def getprice(self):
		return self.__price

	# method
	def printDisk(self):
		print("Disk type : " + str(self.__type))
		print("Disk capacity : " + str(self.__capacity))
		print("Disk price : " + str(self.__price))
	
